src/app/containers/visualizer/visualizer.container.ts

```diff
@@ -289,7 +280,7 @@ export class VisualizerComponent implements OnInit {
         apForm.controls.apDay.enable();
     }
 
-    setF107Values( data: number[], date: moment.Moment ) {
+    setF107Values( data: number[], timestamp: number ) {
```

Comment:
I'm not following why you changed this over to a number now, rather than keeping it as a Moment. It seems like you're calling `moment.utc()` in multiple places now. It seems like it would be better to store the moment and then do the string/number cast once initially if possible?